src/status_im/contexts/wallet/wallet_connect/utils.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im.contexts.wallet.wallet-connect.utils
+  (:require ["@walletconnect/core" :refer [Core]]
+            ["@walletconnect/web3wallet" :refer [Web3Wallet]]
+            [status-im.config :as config]))
+
+(defonce ^:private wallet-connect-metadata
```

Comment:
@briansztamfater @shivekkhurana `defonce` shouldn't be used in a case like this. As far as I've seen, `defonce` is basically a _tool_ to help devs take control in the REPL of when side-effects should happen. A common example in the wild is to initialize global atoms, as in `(defonce foo (atom :bar))`.

There's no side-effect to evaluate `wallet-connect-metadata` (ignoring memory allocation could fail in a parallel universe), hence just `def` is sufficient. In fact, when I see `defonce`, I instantly search for the line of code with side-effects to understand why it's being used.

Reply:
Thanks for the feedback, I added translated strings, so probably `defn` is a better choice than `def` because language could change while the app is active 🤔  wdyt?

Reply:
Good question @briansztamfater. I have the impression our app doesn't react to system language changes and thus, requires a full restart. This means it's okay to use `def` and not worry about reactiveness. Not saying this is the best approach, but it's how I see our app behaving now.

Note: on emulated Android, when the whole system language changes, even after a full app restart it doesn't pick up the correct language and English is still seen everywhere. Maybe this is a problem deserving investigation in the near future.